cinema/views.py

```diff
@@ -34,6 +48,25 @@
     queryset = Movie.objects.all()
     serializer_class = MovieSerializer
 
+    def get_queryset(self):
+        queryset = Movie.objects.all()
+
+        genres = self.request.query_params.get("genres", None)
+        if genres:
+            genres_ids = [int(str_id) for str_id in genres.split(",")]
+            queryset = queryset.filter(genres__id__in=genres_ids)
+
+        actors = self.request.query_params.get("actors", None)
+        if actors:
+            actors_ids = [int(str_id) for str_id in actors.split(",")]
```

Comment:
Consider adding error handling for the conversion of `actors` query parameter to integers. If a non-integer value is passed, it will raise a `ValueError`. You might want to handle this exception to avoid server errors.